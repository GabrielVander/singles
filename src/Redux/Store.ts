import {createStore} from "redux";
import rootReducer from "./Reducers";

class Store {
    static INITIAL_GLOBAL_STATE = {};
    // @ts-ignore
    static store = createStore(rootReducer, Store.INITIAL_GLOBAL_STATE, window.__REDUX_DEVTOOLS_EXTENSION__ && window.__REDUX_DEVTOOLS_EXTENSION__());
}

export default Store;
