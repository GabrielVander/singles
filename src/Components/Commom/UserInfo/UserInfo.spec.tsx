import {render} from "@testing-library/react";
import {Provider} from "react-redux";
import Store from "../../../Redux/Store";
import {BrowserRouter} from "react-router-dom";
import React from "react";
import Userinfo from "./index";

test('matches snapshot', () => {
    const component = render(
        <Provider store={Store.store}>
            <BrowserRouter>
                <Userinfo/>
            </BrowserRouter>
        </Provider>
    );
    expect(component).toMatchSnapshot();
});
