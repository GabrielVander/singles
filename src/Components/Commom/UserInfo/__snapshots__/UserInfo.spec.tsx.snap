// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <form
        action="#"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 dtuOcS"
        >
          <div
            class="StyledBox-sc-13pk1d4-0 dYebPD"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 dLXLoO FormField__FormFieldBox-m9hood-0 jGEMXj"
            >
              <label
                class="StyledText-sc-1sadyjn-0 drTEXR"
              >
                Full name
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 gmReJc FormField__FormFieldContentBox-m9hood-1"
              >
                <div
                  class="StyledTextInput__StyledTextInputContainer-sc-1x30a0s-1 dpaJQp"
                >
                  <input
                    autocomplete="off"
                    class="StyledTextInput-sc-1x30a0s-0 fXMpul"
                    name="fullName"
                    placeholder="Your full name"
                    value=""
                  />
                </div>
              </div>
            </div>
            <div
              class="StyledBox-sc-13pk1d4-0 dLXLoO FormField__FormFieldBox-m9hood-0 jGEMXj"
            >
              <label
                class="StyledText-sc-1sadyjn-0 drTEXR"
              >
                Birthday
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 gmReJc FormField__FormFieldContentBox-m9hood-1"
              >
                <div
                  class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 isRvFr"
                >
                  <div
                    class="StyledMaskedInput__StyledIcon-sc-99vkfa-2 kUaPnG"
                  >
                    <svg
                      aria-label="Calendar"
                      class="StyledIcon-ofa7kd-0 dEeaTZ"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M2,5 L22,5 L22,22 L2,22 L2,5 Z M18,5 L18,1 M6,5 L6,1 M2,10 L22,10"
                        fill="none"
                        stroke="#000"
                        stroke-width="2"
                      />
                    </svg>
                  </div>
                  <input
                    autocomplete="off"
                    class="StyledMaskedInput-sc-99vkfa-0 bpkOdS"
                    name="birthday"
                    placeholder="dd/mm/yyyy"
                    value=""
                  />
                </div>
              </div>
            </div>
            <div
              class="StyledBox-sc-13pk1d4-0 dLXLoO FormField__FormFieldBox-m9hood-0 jGEMXj"
            >
              <label
                class="StyledText-sc-1sadyjn-0 drTEXR"
              >
                How many children?
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 euBdwh FormField__FormFieldContentBox-m9hood-1"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 dvIeIL"
                >
                  <span
                    class="StyledText-sc-1sadyjn-0 DkZvS"
                  >
                    0
                  </span>
                </div>
                <input
                  class="StyledRangeInput-sc-15st9ck-0 heMjld"
                  max="20"
                  min="0"
                  name="children"
                  type="range"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="StyledBox__StyledBoxGap-sc-13pk1d4-1 erOjqd"
          />
          <div
            class="StyledBox-sc-13pk1d4-0 dYebPD"
          >
            <div
              class="StyledBox-sc-13pk1d4-0 dLXLoO FormField__FormFieldBox-m9hood-0 jGEMXj"
            >
              <label
                class="StyledText-sc-1sadyjn-0 drTEXR"
              >
                Country
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 gmReJc FormField__FormFieldContentBox-m9hood-1"
              >
                <button
                  aria-label="Open Drop"
                  class="StyledButton-sc-323bzc-0 hkpJCj Select__StyledSelectDropButton-sc-17idtfo-1 dYfCep"
                  type="button"
                >
                  <div
                    class="StyledBox-sc-13pk1d4-0 jmuHez"
                  >
                    <div
                      class="StyledBox-sc-13pk1d4-0 iAryDB"
                    >
                      <div
                        class="StyledTextInput__StyledTextInputContainer-sc-1x30a0s-1 dpaJQp"
                      >
                        <input
                          autocomplete="off"
                          class="StyledTextInput-sc-1x30a0s-0 fXMpul Select__SelectTextInput-sc-17idtfo-0 bIurki"
                          name="passwordConfirmation"
                          placeholder="Your current country"
                          readonly=""
                          tabindex="-1"
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                    <div
                      class="StyledBox-sc-13pk1d4-0 eA-DBEo"
                      style="min-width: auto;"
                    >
                      <svg
                        aria-label="FormDown"
                        class="StyledIcon-ofa7kd-0 fuviCN"
                        viewBox="0 0 24 24"
                      >
                        <polyline
                          fill="none"
                          points="18 9 12 15 6 9"
                          stroke="#000"
                          stroke-width="2"
                        />
                      </svg>
                    </div>
                  </div>
                </button>
              </div>
            </div>
            <div
              class="StyledBox-sc-13pk1d4-0 dLXLoO FormField__FormFieldBox-m9hood-0 jGEMXj"
            >
              <label
                class="StyledText-sc-1sadyjn-0 drTEXR"
              >
                Gender
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 gmReJc FormField__FormFieldContentBox-m9hood-1"
              >
                <button
                  aria-label="Open Drop"
                  class="StyledButton-sc-323bzc-0 hkpJCj Select__StyledSelectDropButton-sc-17idtfo-1 dYfCep"
                  type="button"
                >
                  <div
                    class="StyledBox-sc-13pk1d4-0 jmuHez"
                  >
                    <div
                      class="StyledBox-sc-13pk1d4-0 iAryDB"
                    >
                      <div
                        class="StyledTextInput__StyledTextInputContainer-sc-1x30a0s-1 dpaJQp"
                      >
                        <input
                          autocomplete="off"
                          class="StyledTextInput-sc-1x30a0s-0 fXMpul Select__SelectTextInput-sc-17idtfo-0 bIurki"
                          name="gender"
                          readonly=""
                          tabindex="-1"
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                    <div
                      class="StyledBox-sc-13pk1d4-0 eA-DBEo"
                      style="min-width: auto;"
                    >
                      <svg
                        aria-label="FormDown"
                        class="StyledIcon-ofa7kd-0 fuviCN"
                        viewBox="0 0 24 24"
                      >
                        <polyline
                          fill="none"
                          points="18 9 12 15 6 9"
                          stroke="#000"
                          stroke-width="2"
                        />
                      </svg>
                    </div>
                  </div>
                </button>
              </div>
            </div>
            <div
              class="StyledBox-sc-13pk1d4-0 dLXLoO FormField__FormFieldBox-m9hood-0 jGEMXj"
            >
              <label
                class="StyledText-sc-1sadyjn-0 drTEXR"
              >
                Languages you speak
              </label>
              <div
                class="StyledBox-sc-13pk1d4-0 gmReJc FormField__FormFieldContentBox-m9hood-1"
              >
                <button
                  aria-label="Open Drop"
                  class="StyledButton-sc-323bzc-0 hkpJCj Select__StyledSelectDropButton-sc-17idtfo-1 dYfCep"
                  type="button"
                >
                  <div
                    class="StyledBox-sc-13pk1d4-0 jmuHez"
                  >
                    <div
                      class="StyledBox-sc-13pk1d4-0 iAryDB"
                    >
                      <div
                        class="StyledTextInput__StyledTextInputContainer-sc-1x30a0s-1 dpaJQp"
                      >
                        <input
                          autocomplete="off"
                          class="StyledTextInput-sc-1x30a0s-0 fXMpul Select__SelectTextInput-sc-17idtfo-0 bIurki"
                          name="languages"
                          readonly=""
                          tabindex="-1"
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                    <div
                      class="StyledBox-sc-13pk1d4-0 eA-DBEo"
                      style="min-width: auto;"
                    >
                      <svg
                        aria-label="FormDown"
                        class="StyledIcon-ofa7kd-0 fuviCN"
                        viewBox="0 0 24 24"
                      >
                        <polyline
                          fill="none"
                          points="18 9 12 15 6 9"
                          stroke="#000"
                          stroke-width="2"
                        />
                      </svg>
                    </div>
                  </div>
                </button>
              </div>
            </div>
          </div>
        </div>
        <div
          class="StyledBox-sc-13pk1d4-0 cqyhTO"
        >
          <button
            class="StyledButton-sc-323bzc-0 hmOcFG"
            type="submit"
          >
            Save
          </button>
        </div>
      </form>
    </div>
  </body>,
  "container": <div>
    <form
      action="#"
    >
      <div
        class="StyledBox-sc-13pk1d4-0 dtuOcS"
      >
        <div
          class="StyledBox-sc-13pk1d4-0 dYebPD"
        >
          <div
            class="StyledBox-sc-13pk1d4-0 dLXLoO FormField__FormFieldBox-m9hood-0 jGEMXj"
          >
            <label
              class="StyledText-sc-1sadyjn-0 drTEXR"
            >
              Full name
            </label>
            <div
              class="StyledBox-sc-13pk1d4-0 gmReJc FormField__FormFieldContentBox-m9hood-1"
            >
              <div
                class="StyledTextInput__StyledTextInputContainer-sc-1x30a0s-1 dpaJQp"
              >
                <input
                  autocomplete="off"
                  class="StyledTextInput-sc-1x30a0s-0 fXMpul"
                  name="fullName"
                  placeholder="Your full name"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 dLXLoO FormField__FormFieldBox-m9hood-0 jGEMXj"
          >
            <label
              class="StyledText-sc-1sadyjn-0 drTEXR"
            >
              Birthday
            </label>
            <div
              class="StyledBox-sc-13pk1d4-0 gmReJc FormField__FormFieldContentBox-m9hood-1"
            >
              <div
                class="StyledMaskedInput__StyledMaskedInputContainer-sc-99vkfa-1 isRvFr"
              >
                <div
                  class="StyledMaskedInput__StyledIcon-sc-99vkfa-2 kUaPnG"
                >
                  <svg
                    aria-label="Calendar"
                    class="StyledIcon-ofa7kd-0 dEeaTZ"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M2,5 L22,5 L22,22 L2,22 L2,5 Z M18,5 L18,1 M6,5 L6,1 M2,10 L22,10"
                      fill="none"
                      stroke="#000"
                      stroke-width="2"
                    />
                  </svg>
                </div>
                <input
                  autocomplete="off"
                  class="StyledMaskedInput-sc-99vkfa-0 bpkOdS"
                  name="birthday"
                  placeholder="dd/mm/yyyy"
                  value=""
                />
              </div>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 dLXLoO FormField__FormFieldBox-m9hood-0 jGEMXj"
          >
            <label
              class="StyledText-sc-1sadyjn-0 drTEXR"
            >
              How many children?
            </label>
            <div
              class="StyledBox-sc-13pk1d4-0 euBdwh FormField__FormFieldContentBox-m9hood-1"
            >
              <div
                class="StyledBox-sc-13pk1d4-0 dvIeIL"
              >
                <span
                  class="StyledText-sc-1sadyjn-0 DkZvS"
                >
                  0
                </span>
              </div>
              <input
                class="StyledRangeInput-sc-15st9ck-0 heMjld"
                max="20"
                min="0"
                name="children"
                type="range"
                value=""
              />
            </div>
          </div>
        </div>
        <div
          class="StyledBox__StyledBoxGap-sc-13pk1d4-1 erOjqd"
        />
        <div
          class="StyledBox-sc-13pk1d4-0 dYebPD"
        >
          <div
            class="StyledBox-sc-13pk1d4-0 dLXLoO FormField__FormFieldBox-m9hood-0 jGEMXj"
          >
            <label
              class="StyledText-sc-1sadyjn-0 drTEXR"
            >
              Country
            </label>
            <div
              class="StyledBox-sc-13pk1d4-0 gmReJc FormField__FormFieldContentBox-m9hood-1"
            >
              <button
                aria-label="Open Drop"
                class="StyledButton-sc-323bzc-0 hkpJCj Select__StyledSelectDropButton-sc-17idtfo-1 dYfCep"
                type="button"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 jmuHez"
                >
                  <div
                    class="StyledBox-sc-13pk1d4-0 iAryDB"
                  >
                    <div
                      class="StyledTextInput__StyledTextInputContainer-sc-1x30a0s-1 dpaJQp"
                    >
                      <input
                        autocomplete="off"
                        class="StyledTextInput-sc-1x30a0s-0 fXMpul Select__SelectTextInput-sc-17idtfo-0 bIurki"
                        name="passwordConfirmation"
                        placeholder="Your current country"
                        readonly=""
                        tabindex="-1"
                        type="text"
                        value=""
                      />
                    </div>
                  </div>
                  <div
                    class="StyledBox-sc-13pk1d4-0 eA-DBEo"
                    style="min-width: auto;"
                  >
                    <svg
                      aria-label="FormDown"
                      class="StyledIcon-ofa7kd-0 fuviCN"
                      viewBox="0 0 24 24"
                    >
                      <polyline
                        fill="none"
                        points="18 9 12 15 6 9"
                        stroke="#000"
                        stroke-width="2"
                      />
                    </svg>
                  </div>
                </div>
              </button>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 dLXLoO FormField__FormFieldBox-m9hood-0 jGEMXj"
          >
            <label
              class="StyledText-sc-1sadyjn-0 drTEXR"
            >
              Gender
            </label>
            <div
              class="StyledBox-sc-13pk1d4-0 gmReJc FormField__FormFieldContentBox-m9hood-1"
            >
              <button
                aria-label="Open Drop"
                class="StyledButton-sc-323bzc-0 hkpJCj Select__StyledSelectDropButton-sc-17idtfo-1 dYfCep"
                type="button"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 jmuHez"
                >
                  <div
                    class="StyledBox-sc-13pk1d4-0 iAryDB"
                  >
                    <div
                      class="StyledTextInput__StyledTextInputContainer-sc-1x30a0s-1 dpaJQp"
                    >
                      <input
                        autocomplete="off"
                        class="StyledTextInput-sc-1x30a0s-0 fXMpul Select__SelectTextInput-sc-17idtfo-0 bIurki"
                        name="gender"
                        readonly=""
                        tabindex="-1"
                        type="text"
                        value=""
                      />
                    </div>
                  </div>
                  <div
                    class="StyledBox-sc-13pk1d4-0 eA-DBEo"
                    style="min-width: auto;"
                  >
                    <svg
                      aria-label="FormDown"
                      class="StyledIcon-ofa7kd-0 fuviCN"
                      viewBox="0 0 24 24"
                    >
                      <polyline
                        fill="none"
                        points="18 9 12 15 6 9"
                        stroke="#000"
                        stroke-width="2"
                      />
                    </svg>
                  </div>
                </div>
              </button>
            </div>
          </div>
          <div
            class="StyledBox-sc-13pk1d4-0 dLXLoO FormField__FormFieldBox-m9hood-0 jGEMXj"
          >
            <label
              class="StyledText-sc-1sadyjn-0 drTEXR"
            >
              Languages you speak
            </label>
            <div
              class="StyledBox-sc-13pk1d4-0 gmReJc FormField__FormFieldContentBox-m9hood-1"
            >
              <button
                aria-label="Open Drop"
                class="StyledButton-sc-323bzc-0 hkpJCj Select__StyledSelectDropButton-sc-17idtfo-1 dYfCep"
                type="button"
              >
                <div
                  class="StyledBox-sc-13pk1d4-0 jmuHez"
                >
                  <div
                    class="StyledBox-sc-13pk1d4-0 iAryDB"
                  >
                    <div
                      class="StyledTextInput__StyledTextInputContainer-sc-1x30a0s-1 dpaJQp"
                    >
                      <input
                        autocomplete="off"
                        class="StyledTextInput-sc-1x30a0s-0 fXMpul Select__SelectTextInput-sc-17idtfo-0 bIurki"
                        name="languages"
                        readonly=""
                        tabindex="-1"
                        type="text"
                        value=""
                      />
                    </div>
                  </div>
                  <div
                    class="StyledBox-sc-13pk1d4-0 eA-DBEo"
                    style="min-width: auto;"
                  >
                    <svg
                      aria-label="FormDown"
                      class="StyledIcon-ofa7kd-0 fuviCN"
                      viewBox="0 0 24 24"
                    >
                      <polyline
                        fill="none"
                        points="18 9 12 15 6 9"
                        stroke="#000"
                        stroke-width="2"
                      />
                    </svg>
                  </div>
                </div>
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        class="StyledBox-sc-13pk1d4-0 cqyhTO"
      >
        <button
          class="StyledButton-sc-323bzc-0 hmOcFG"
          type="submit"
        >
          Save
        </button>
      </div>
    </form>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
